SPECS/environment-modules/environment-modules.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "modules-5.2.0.tar.bz2": "65f6537e770c66036c8adcad57c51a389f805d19fe942c963bd9f29337832b6e"
```

Comment:
As part of this change, let's also re-name the tarball, so it's clearer what they are for at the same reducing a risk of conflict. I'd suggest the good old `%{name}-%{version}.tar.bz2`.;)

Reply:
updated